pyVor/tests/testPrimitives.py

```diff
@@ -271,6 +271,11 @@ def test_to_vector(self):
         self.assertEqual(self.b.to_vector(), Vector(1, 2))
         self.assertNotEqual(self.b.to_vector(), Vector(1, 2, 0))
 
+    def test_to_array(self):
```

Comment:
Yeah. Well mostly that happened because I was gonna make tests for it and then I realized it shouldn't exist. So that's just there as documentation.